recipes/superlu_dist/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = "5.2.2" %}
```

Comment:
Why use an old version?

Reply:
No reason I guess, we could use the latest (6.1.1).